### PR TITLE
dw/rb - changed favicon s21-6pm-4

### PR DESCRIPTION
**Pull request by dorinori**
In this PR, we updated the favicon so that the user can see a cuter favicon.

### CODE REVIEW

File: secrets-localhost.properties.SAMPLE [hunk 1]
```diff
@@ -2,7 +2,7 @@ app.namespace=https://YOUR-HEROKU-APP.herokuapp.com
 app.admin.emails=phtcon@ucsb.edu
 app.member.hosted-domain=ucsb.edu
 
-auth0.domain=YOUR-AUTH0-TENANT.us.auth0.com
+auth0.domain=cs156-s21-6pm-4.us.auth0.com
```
Comment (WadeVaresio) [inline secrets-localhost.properties.SAMPLE:5]: As this is a sample file, we need to keep the default values so users can edit them later on for their purposes. Please revert this back to the original default value.

**Review by WadeVaresio — CHANGES_REQUESTED** [review]
All looks good except for the secrets-localhost.properties.SAMPLE file. Please revert the file back to its default state.

**Review by WadeVaresio — APPROVED** [review]
LGTM

**Review by bzamora020 — APPROVED** [review]
LGTM!

**Review by samliangsk — APPROVED** [review]
LGTM

**Pull request merged**